packages/dashboard/src/testing/testing-ground/testing-ground.tsx

```diff
@@ -43,6 +160,57 @@ export class TestingGround {
     this.stretchToFit = (e as any).target.checked;
   };
 
+  addAnnotation = (e: Event) => {
+    e.preventDefault();
+    const original = this.dashboardConfiguration;
```

Comment:
no need to declare this variable, you can just use `dashboardConfiguration: this.dashboardConfiguration,` in the object.

Reply:
Removed!